packages/frontend/src/store/tasks.ts

```diff
@@ -58,47 +58,34 @@ const mutations = {
 
 const actions = {
   FETCH_TASKS: async ({ commit }: ActionContext<State, State>) => {
-    try {
-      const url = new URL(config.API_HOST + "/api/user/tasks").toString();
-      const res = await adAuthenticatedFetch(url);
-      if (res.status === 404) {
-        commit("SET_USER_NOT_FOUND");
-        throw res.statusText;
-      }
-      const tasks = await res.json();
-      commit("SET_TASKS", tasks);
-    } catch (e) {
-      if (e !== "Not Found") {
+    httpClient
+      .get(`${config.API_HOST}/api/user/tasks`)
+      .then(response => {
+        commit("SET_TASKS", response.data);
+      })
+      .catch(e => {
         console.error(e);
         commit("ADD_TO_ERROR_LIST", e);
-      }
-    }
+      });
   },
 
   PUSH_TASKS: async (
```

Comment:
Denne trenger ikke `async`-keywordet hvis den ikke skal bruke `async/await`, det holder at den bruker `Promise.then`. Evt. kan den skrives om til å bruke `try/catch` rundt `async/await`-kall.

```suggestion
  PUSH_TASKS: (
```